.github/pull_request_template.md

```diff
@@ -1,3 +1,10 @@
+<!--  Thanks for sending a pull request, here are some tips:
```

Comment:
Does the `<!--` comment syntax mean this text is only visible to a user while editing the PR description?

Reply:
Yes, it is not visible during the markdown rendering